src/Illuminate/Support/Facades/URL.php

```diff
@@ -15,7 +15,7 @@
  * @method static string signedRoute(string $name, array $parameters = [], \DateTimeInterface|\DateInterval|int $expiration = null, bool $absolute = true)
  * @method static string temporarySignedRoute(string $name, \DateTimeInterface|\DateInterval|int $expiration, array $parameters = [], bool $absolute = true)
  * @method static string hasValidSignature(\Illuminate\Http\Request $request, bool $absolute = true)
- * @method static void defaults(array $defaults)
+ * @method static void defaults(array $defaults, string $namePrefix = null)
```

Comment:
`string|null` instead of `string` please

Reply:
Thanks! :D